test/master-playlist-controller.test.js

```diff
@@ -573,8 +577,8 @@ QUnit.test('updates the enabled track when switching audio groups', function(ass
                                 '0.ts\n' +
                                 '#EXT-X-ENDLIST\n');
 
-  assert.ok(mpc.activeAudioGroup().filter((track) => track.enabled)[0],
-           'enabled a track in the new audio group');
+  assert.notOk(mpc.mediaTypes_.AUDIO.activePlaylistLoader,
```

Comment:
It is switching from unmuxed to muxed audio. When we are in the muxed audio case, there is no active audio playlist loader because the audio content is coming from the video stream